kclvm/compiler_base/error/src/diagnostic/mod.rs

```diff
@@ -1 +1,109 @@
+use self::style::DiagnosticStyle;
+use rustc_errors::styled_buffer::StyledBuffer;
+
+pub mod pendant;
 pub mod style;
+
+#[cfg(test)]
+mod tests;
+
```

Comment:
It is recommended to illustrate the relationship between structures/traits of your core design through UML diagrams.